networking/configuring-a-custom-pki.adoc

```diff
@@ -0,0 +1,56 @@
+[id="configuring-a-custom-pki"]
+= Configuring a custom PKI
+include::modules/common-attributes.adoc[]
+:context: configuring-a-custom-pki
+
+toc::[]
+
+Some platform components (i.e., web console) use Routes for communication and,
+therefore, must trust other components' certificates to interact with them. If
+you're using a custom public key infrastructure (PKI), you must configure it so
+its privately signed CA certificates are recognized across the cluster.
+
+You can leverage the Proxy API to configure your custom PKI. You must do this
+either during installation *or* at runtime.
+
+* During *installation*, configure the cluster-wide proxy. You must define your
+privately signed CA certificates in the `install-config.yaml` file's
+`additionalTrustBundle` setting.
++
+The installation program generates a ConfigMap that is named `user-ca-bundle`
+that contains the additional CA certificates you defined. The Cluster Network
+Operator then creates a `trusted-ca-bundle` ConfigMap that merges these contents
+with the {op-system-first} trust bundle; this ConfigMap is referenced in the
+Proxy object's `trustedCA` field.
+
+* At *runtime*, enable the cluster-wide proxy. This involves creating a
```

Comment:
Regarding "enable the cluster-wide proxy", 4.2 and greater clusters will always have a proxy object named "cluster". If proxy is not enabled at install time, the object will have a nil spec and status. A user is not necessarily enabling proxy. Instead, the `trustedCA` field of proxy "cluster" is used to add additional cluster-wide trusted CA's.